### PR TITLE
Add unit tests.

### LINKED ISSUE #13: Add unit tests in addition to integration tests
Right now, we are testing via integration tests, which is a start, but it would be beneficial to test the core business logic via unit tests as well

### PR DESCRIPTION
**Pull request by jezdez**
Resolves #13

### CODE REVIEW

File: anaconda_anon_usage/patch.py [hunk 1]
```diff
@@ -80,3 +80,5 @@ def _new_init(*args, **kwargs):
             _old__init__(*args, **kwargs)
 
         context.__init__ = _new_init
+
+    return True
```
Comment (jezdez) [inline anaconda_anon_usage/patch.py:84]: Just returning if the patch was done or not

**Review by mcg1969 — APPROVED** [review]

**Comment by mcg1969** [comment]
Looks good to me. If those tests pass we're good

File: anaconda_anon_usage/tokens.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 from .utils import _debug, _random_token, _saved_token
 
 Tokens = namedtuple("Tokens", ("version", "client", "session", "environment"))
+CONFIG_DIR = expanduser("~/.conda")
```
Comment (jezdez) [inline anaconda_anon_usage/tokens.py:16]: I'm wondering if we should use something like https://github.com/platformdirs/platformdirs to enable roaming profiles on Windows etc?

Comment (mcg1969) [reply anaconda_anon_usage/tokens.py:16]: Yes, maybe create a separate issue to duplicate conda's own conventions for this directory.

**Pull request merged**